.env

```diff
@@ -1,3 +1,4 @@
-REACT_APP_CHAIN_ID="1"
+REACT_APP_CHAIN_ID="3"
 REACT_APP_GOOGLE_ANALYTICS_MEASUREMENT_ID=""
 REACT_APP_NETWORK_URL="http://127.0.0.1:8545/"
+REACT_APP_NOTIFY_DAPP_ID="3f1ffee0-a39b-4f4e-8371-f0e157e4172e"
```

Comment:
just upgraded to a paid blocknative plan.  @sandrasong we will need to rotate this key out for a new one with restricted origins, please ping me on slack before landing this PR and we'll coordinate!